java/java-fundamentals/exceptions-i/throwing-an-exception.md

```diff
@@ -4,31 +4,35 @@ type: normal
 category: must-know
 links:
   - >-
-    [crunchify.com](http://crunchify.com/better-understanding-on-checked-vs-unchecked-exceptions-how-to-handle-exception-better-way-in-java/){website}
+    [How to Handle Exceptions Better](http://crunchify.com/better-understanding-on-checked-vs-unchecked-exceptions-how-to-handle-exception-better-way-in-java/){website}
 notes: |
   https://insights.enki.com/#/contrib/55a3b45d2a4a89320042e651
+revisionQuestion:
+  formats:
+    - fill-in-the-gap
+    - type-in-the-gap
+  context: standalone
 ---
 
 # Throwing an `exception`
 
-
 ---
 
 ## Content
 
-An `exception` is an event, which occurs during the execution of a program, that disrupts the normal flow of the program's instructions.
+An `exception` is an event that occurs during the execution of a program disrupting the normal flow of the program.
```

Comment:
We should change `exception` to _exception_ in this whole insight because "exception" is not a keyword

Reply:
```suggestion
An *exception* is an event that occurs during the execution of a program disrupting the normal flow of the program.
```